integration_tests/CMakeLists.txt

```diff
@@ -2,11 +2,14 @@
 include_directories(
     SYSTEM ${CMAKE_SOURCE_DIR}/third_party/gtest/googletest/include
     SYSTEM ${CMAKE_SOURCE_DIR}/third_party/gtest/googlemock/include
+    SYSTEM ${CMAKE_SOURCE_DIR}/third_party/mavlink/include
 )
 
 # These are consumed in ../CMakeLists.txt
 set(integration_tests_src ${integration_tests_src} PARENT_SCOPE)
 
+file(GLOB_RECURSE integration_tests_src "*.cpp")
```

Comment:
I think the good practice is to list the files, instead of doing a `GLOB_RECURSE`?!

Reply:
Yes that's what I read too.

Reply:
I see, I could see that here: https://cmake.org/cmake/help/v3.9/command/file.html

Note: *We do not recommend using GLOB to collect a list of source files from your source tree.
If no CMakeLists.txt file changes when a source is added or removed
then the generated build system cannot know when to ask CMake to regenerate.*